code/addons/actions/template/stories/spies.stories.ts

```diff
@@ -0,0 +1,24 @@
+import { global as globalThis } from '@storybook/global';
+import { spyOn } from '@storybook/test';
+
+export default {
+  component: globalThis.Components.Button,
+  loaders() {
+    spyOn(console, 'log').mockName('console.log');
+  },
+  args: {
+    label: 'Button',
+  },
+  parameters: {
+    chromatic: { disable: true },
+  },
+};
+
+export const ShowSpyOnInActions = {
+  args: {
+    onClick: () => {
+      console.log('first');
+      console.log('second');
+    },
+  },
```

Comment:
maybe also log in a play-function, just to be sure?

Reply:
I think it is an overkill, as it is now clicked in the e2e tests, which is kind of happening in the same phase as the `play` function.